skopt/plots.py

```diff
@@ -88,3 +90,245 @@ def plot_convergence(*args, **kwargs):
         ax.legend(loc="best")
 
     return ax
+
+
+def _format_scatter_plot_axes(ax, space, ylabel):
+    # Deal with formatting of the axes
+    for i in range(space.n_dims):
+        for j in range(space.n_dims):
+            ax_ = ax[i, j]
+
+            if j > i:
+                ax_.axis("off")
+
+            # adjust bounds for every off-diagonal axis
+            if i != j:
+                ax_.set_ylim(*space.dimensions[i].bounds)
+                ax_.set_xlim(*space.dimensions[j].bounds)
+                # keep y axis tick labels for the diagonal
+                if j > 0:
+                    ax_.set_yticklabels([])
+            else:
+                ax_.yaxis.tick_right()
+                ax_.yaxis.set_label_position('right')
+                ax_.yaxis.set_ticks_position('both')
+                ax_.set_ylabel(ylabel)
+
+            ax_.xaxis.set_major_locator(MaxNLocator(6, prune='both'))
+            ax_.yaxis.set_major_locator(MaxNLocator(6, prune='both'))
+
+            if i < space.n_dims - 1:
+                ax_.set_xticklabels([])
+            # bottom row
+            else:
+                [l.set_rotation(45) for l in ax_.get_xticklabels()]
+
+    return ax
+
+
+def partial_dependence(space, model, i, j=None, sample_points=None,
+                       n_samples=100, n_points=40):
+    """Calculate partial dependence of `model` for dimensions `i` and `j`
```

Comment:
Maybe instead something like "Calculate the partial dependence for dimensions `i` and `j` with respect to the objective value".